[Focus Mode] Images and content for the security tool

captainbrosset opened the pull request:
New Focus Mode images and content for the **Understand security issues using the Security tool** article.

Before: https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/
After: https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/index?branch=pr-en-us-2939

AB#47818857

mikehoffms commented on microsoft-edge/devtools-guide-chromium/security/index.md:
```diff
@@ -57,29 +57,33 @@ If you haven't set up HTTPS on your server, [Let's Encrypt](https://letsencrypt.
 
 
 <!-- ====================================================================== -->
 ## View details
```
i consider non-blocking and arguably out of scope for this PR:
* h2 followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)

mikehoffms commented on microsoft-edge/devtools-guide-chromium/security/index.md:
```diff
@@ -44,7 +44,7 @@ Use the **Security** panel to inspect the security of a page.
 
```
i consider non-blocking and arguably out of scope for this PR:
* h2 ("Common problems") followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/?branch=pr-en-us-2939#common-problems

mikehoffms commented on microsoft-edge/devtools-guide-chromium/security/index.md:
imo, 3 pixels width of red box line would look better than 5 pixels.   
5 = chunky, disproportionately thick.  
3 = subtle and best for an overlay of UI.

mikehoffms reviewed (approved):
Writer/Editor pass.  
* Requested red boxes 3 px instead of 5 px width of red border line.  
* Noted existing lack of text between h2s & h3s, against style guides (out of scope b/c not text or pngs about F.M.)

captainbrosset commented:
> * Requested red boxes 3 px instead of 5 px width of red border line.

Done

> * Noted existing lack of text between h2s & h3s, against style guides (out of scope b/c not text or pngs about F.M.)

I converted h3s to h4s, and added intro text beneath h2s.
